bin/commands/install.js

```diff
@@ -1,12 +1,29 @@
-var
-  childProcess = require('child_process');
+/* eslint-disable no-console */
 
-// "kuzzle install" is an alias for "kuzzle plugins --install"
+var
+  params = require('rc')('kuzzle'),
+  Kuzzle = require('../../lib/api'),
+  RequestObject = require('kuzzle-common-objects').Models.requestObject;
 
 module.exports = function () {
-  childProcess
-    .spawn(require.main.filename, ['plugins', '--install'], {stdio: 'inherit'})
-    .on('close', code => {
-      process.exit(code);
+  var 
+    kuzzle = new Kuzzle(),
+    requestObject = new RequestObject({
+      body: {install: true}
+    });
+
+  console.log('███ kuzzle-plugins: Starting plugins installation...');
+  
+  kuzzle.start(params, {dummy: true})
```

Comment:
Actually, went a bit too fast. I reverted my changes.

The thing is that this command is run *before* kuzzle actually starts. It is triggered from the run*.sh scripts: 
```
node bin/kuzzle install && pm2 start /config/processes-dev.json
```

So, we need the controllers, the pluginsManager and the InternalEngine but **not** the internalbroker client.
The dummy start just does what we need.

Because we don't have a running Kuzzle to talk to, we cannot actually use the remoteActions.do method either.